posthog/hogql_queries/insights/trends/trends_query_runner.py

```diff
@@ -213,60 +213,65 @@ def to_actors_query_options(self) -> InsightActorsQueryOptionsResponse:
             ]
 
         # Breakdowns
-        for series in self.query.series:
-            # TODO: Add support for DataWarehouseNode
-            if isinstance(series, DataWarehouseNode):
-                continue
+        if self.query.breakdownFilter is not None:
+            res_breakdown = []
+            if self.query.breakdownFilter.breakdown_type == "cohort":
+                assert isinstance(self.query.breakdownFilter.breakdown, list)
+                for value in self.query.breakdownFilter.breakdown:
+                    if value != "all" and str(value) != "0":
+                        res_breakdown.append(
+                            BreakdownItem(label=Cohort.objects.get(pk=int(value), team=self.team).name, value=value)
+                        )
+                    else:
+                        res_breakdown.append(BreakdownItem(label="all users", value="all"))
+            else:
+                # TODO: Work out if we will have issues only getting breakdown values for
```

Comment:
What will happen here if breakdowns come out different for current and previous?

This change could make them have different breakdown values (or intervals), and group different things into other, couldn't it?

Maybe we could run current and previous in the same query with larger time bounds instead of in two queries.

But similarly to compare, how does this work for any query with multiple series in it (I'm not terribly familiar with that use case, I just know it exists)

Reply:
![image](https://github.com/PostHog/posthog/assets/1855120/44778306-f6e2-4079-8036-ba3f6c2a47e4)

It works fine, both series just show different values, so not the most useful. This wasn't changed from the prior behavior, so is probably okay!